### ISSUE TITLE
"Read 2 bytes failed" when tryng to dump 0xF1

### ISSUE THREAD

**Issue by DDevil97** [oranav/i9300_emmc_toolbox#13]
Hello.
Im trying to revive a bricked i9300, but i cannot dump the eMMC firmware, it starts and shows text on the phone screen, but gets stuck saying "Read 2 bytes failed" (i have --verbose in the parameters).
The "hello world" works, also dumping the SBOOT. I have the UBUGNK1 bootloader.
I can describe more if someone can help me.

**Comment by macaquinyho** [comment]
https://theandroid02.github.io/i9300-EMMC-GUIDE/

Try my guide if you want, I have a 0xF7 dump there.
Any problem tell me

**Comment by DDevil97** [comment]
Hello thank you very much!
I'm still having problems, i followed all the steps for the three diferent types of bricks, i will describe what the phone does: If i just try to turn it on, it tooks like 15 seconds to "turn on", and then just a blank screen or colored white noise apears on the screen. Booting into download mode does the same thing.
If i try to boot from sd card (volume down+home+power and then volume up) the same thing happends. Whenever i try to upload the eMMC firmware with the write_fw.bin shellcode or read the firmware on the phone eMMC with the dump_fw_bootrom shellcode, it just gets going forever saying "DEBUG:root:Read 2 bytes failed" until i disconect the phone (the "shellcode started" apears on the phone screen and the console, the "booted emmc into bootrom mode" message only apears after a while in the phone screen, the console ust keeps saying the read error). And the most interesting thing, if i try to read the phone's eMMC firmware with the dump_fw.bin shellcode or if i try to rezsize the boot partition with the change_boot_partition_size.bin shellcode, the program terminates with error code -19 inmmediately and the phone turns with a non-functional download mode, like the normal one but without the custom firmware count and such legends on the upper corner. Then it behaves exactly the same as when the screen is blank/noisiy. EDIT: All of this happends whatever i have the sd card on the phone or not.

**Comment by macaquinyho** [comment]
So, if I understand this correctly, your phone even with no sdcard boots to the white "corrupted" screen?

**Comment by DDevil97** [comment]
Exactly! and is the only thing it will do apart from that "fake" download mode

**Comment by macaquinyho** [comment]
If you boot with the sdcard and dump sboot, what version does it give you?

**Comment by DDevil97** [comment]
Wait a moment please!

**Comment by DDevil97** [comment]
No luck, the same as before, UBUGNK1, seems like isnt booting from the sd

**Comment by macaquinyho** [comment]
I have an idea, at this point just email me at ggmaget@gmail.com or join this telegram group: @S3bois

**Comment by oranav** [comment]
@DDevil97 As I see it there are two options. Either your eMMC is bricked by a different bug (maybe a hardware issue) and therefore is unrestorable using my method -- this explains error -19 (eMMC timeout), or that my exploit isn't compatible with your bootloader, therefore its heuristics cannot find the correct functions to execute. Had you had any luck using the type 3 guide (resistor short)? Or any progress with @theandroid02?

**Comment by macaquinyho** [comment]
@oranav, he told me he booted XXELLA with the resistor method, but i havent heard of him since. 

Oh and months ago i unbricked mine that had a 4.0.4 bootloader ( incompatible) with type 3.

**Comment by DDevil97** [comment]
Sorry for dissapearing, i gave the phone dead for good, i wasn't able to boot to XXELLA , the resistor method definitively boot into the "boot from sd card" menu, but every time it failed, as far as i know, the phone is unable to write to the eMMc, probabily due to hardware failure as oranv says. In these months i made several attemps with no luck, any discoveries or something i may find i will post here, thanks for all the help this community is amazing!
